programs/bpf_loader/src/lib.rs

```diff
@@ -1336,6 +1336,7 @@ fn process_loader_upgradeable_instruction(
                 additional_bytes
             );
         }
+        UpgradeableLoaderInstruction::Migrate => unimplemented!(),
```

Comment:
This should return `InstructionError::InvalidInstructionData`. Can't merge any code into master that would cause a validator to panic

Reply:
Makes sense, done with 8eed5abc8f